content/index.md

```diff
@@ -82,38 +99,30 @@ Note that this also means authors need basic familiarity with JavaScript and a c
 See [Getting Started](#getting-started) for integration with server-side frameworks.
 
 
-Getting Started <span id="getting-started"></span>
----------------
+<div class="icon-heading">
+    <div class="badge">
+        <svg class="icon" role="presentation">
+            <use href="#icon-getting-started" xlink:href="#icon-getting-started"></use>
+        </svg>
+    </div>
+    <h2 id="components">Getting Started</h2>
+</div>
 
 We currently have support for complate for developing components in a styleguide and for porting components accross several different platforms.
 [View this guide](getting-started.html) to see all of the different options that we have available.
 
 
-Glossary
---------
-
-<dl>
-<dt id="macro">macro</dt>
-<dd>
-
-complate components -- specifically their JavaScript implementation -- are called macros because they typically expand to result in more complex markup structures.
-
-</dd>
-
-<dt id="progressive-rendering">progressive rendering</dt>
-<dd>
-
-HTTP and HTML allow [streaming pieces of content as soon as possible](https://medium.com/ben-and-dion/progressive-rendering-a-killer-and-under-appreciated-feature-of-the-web-97c789b608c1), rather than waiting for the entire page to finish rendering before serving it.
-
-</dd>
-
-<dt id="universal-rendering">universal rendering</dt>
-<dd>
-
-Sometimes also known as "isomorphic JavaScript", this means using the same components (macros) on both the server and the client.
+<div class="icon-heading">
+    <div class="badge">
+        <svg class="icon" role="presentation">
+            <use href="#icon-faq" xlink:href="#icon-faq"></use>
+        </svg>
+    </div>
+    <h2 id="components">Frequently Asked Questions</h2>
```

Comment:
```suggestion
    <h2>Frequently Asked Questions</h2>
```

(Though it'd be nice if all sections had an ID for deep linking, even if it's not currently exposed via the UI.)